### ISSUE TITLE
host-only-cidr option is ignored

### ISSUE THREAD

**Issue by kameshsampath** [minishift/minishift#686]
as per the minishift docs it said that it supports `host-only-cidr` option with `virtualbox` driver. when we set it via config say `minshift config set host-only-cide="192.168.33.1/24"` or pass it via start command `minshift  start --host-only-cide="192.168.33.1/24"` both these options does not seem to have any effect as minishift seem to use the `192.168.99.1/24` only.

**Comment by LalatenduMohanty** [comment]
I can reproduce this.

```
$ minishift start --vm-driver=virtualbox --host-only-cidr 192.168.99.101/24
Starting local OpenShift cluster using 'virtualbox' hypervisor...
Downloading ISO 'https://github.com/minishift/minishift-b2d-iso/releases/download/v1.0.2/minishift-b2d.iso'
 40.00 MB / 40.00 MB [==========================================================================================================================================================================] 100.00% 0s
 18.93 MB / 18.93 MB [==========================================================================================================================================================================] 100.00% 0s
-- Checking OpenShift client ... OK
-- Checking Docker client ... OK
-- Checking Docker version ... OK
-- Checking for existing OpenShift container ... OK
-- Checking for openshift/origin:v1.5.0-rc.0 image ... 
   Pulling image openshift/origin:v1.5.0-rc.0
   Pulled 0/3 layers, 3% complete
   Pulled 1/3 layers, 70% complete
   Pulled 2/3 layers, 77% complete
   Pulled 3/3 layers, 100% complete
   Extracting
   Image pull complete
-- Checking Docker daemon configuration ... OK
-- Checking for available ports ... OK
-- Checking type of volume mount ... 
   Using Docker shared volumes for OpenShift volumes
-- Creating host directories ... OK
-- Finding server IP ... 
   Using 192.168.99.100 as the server IP
-- Starting OpenShift container ... 
   Creating initial OpenShift configuration
   Starting OpenShift using container 'origin'
   Waiting for API server to start listening
   OpenShift server started
-- Adding default OAuthClient redirect URIs ... OK
-- Installing registry ... OK
-- Installing router ... OK
-- Importing image streams ... OK
-- Importing templates ... OK
-- Login to server ... OK
-- Creating initial project "myproject" ... OK
-- Removing temporary directory ... OK
-- Checking container networking ... OK
-- Server Information ... 
   OpenShift server started.
   The server is accessible via web console at:
       https://192.168.99.100:8443

   You are logged in as:
       User:     developer
       Password: developer

   To login as administrator:
       oc login -u system:admin

$ minishift ip
192.168.99.100


```

**Comment by hferentschik** [comment]
Looks like we might need to get the latest docker-machine version - https://github.com/kubernetes/minikube/issues/510

**Comment by hferentschik** [comment]
This is actually fixed by pull request #773 as well

**Comment by hferentschik** [comment]
Merged via pull request #773
